_data/poems/is-5/one-xxviii.yaml

```
@@ -1,3 +1,24 @@
 title: ONE XXVIII
 
 first_line: my uncle
+
+text: |-1
+ my uncle
+ Daniel fought in the civil
+ war band and can play the triangle
+ like the devil)my
+
+ uncle Frank has done nothing for many
+ years but fly kites and
+ when the
+ string breaks(or something)my uncle Frank breaks into
+ tears. my uncle Tom
+
+ knits and is a kewpie above the cars(but
+ 
+ my uncle Ed
+ that's
+ dead from the neck
+
+ up is lead all over
+ Brattle street by a castrated pup
```

```suggestion
 Brattle Street by a castrated pup
```
Street is uppercase in the Firmage edition